src/procedures/migrate-zigbee-zha.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: getting-started
+description: Home Assistant SkyConnect
+guideName: Migrate a Zigbee2MQTT network using ZHA
+
+---
+<div class="container">
+    <p class="text-center note">Follow this guide if you have a Zigbee2MQTT network running and want to migrate to SkyConnect and ZHA.</p>
+    <p class="text-center note">Note: this procedure helps migrating the network. This means you won't have to pair all your devices again with Home Assistant SkyConnect. However, some of the higher-level settings cannot be migrated. Elements such as device names may be lost. There is currently no migration path to migrate all the settings.</p>
```

Comment:
I was hesitant to call this a "migration" for this specific reason: not all devices will migrate, and those that do will not migrate immediately! When someone actually follows this procedure, they likely will be greeted with an empty ZHA setup that has no devices other than the coordinator.

This migration path will only allow ZHA to use the same network settings as Z2M and "take over" the network: ZHA won't be aware of any other devices on the network and will discover them one-by-one as they check in (if they check in). This will only really work for mains-powered devices like lightbulbs, outlets, etc., that don't sleep.

One possible re-wording to clarify this:

```suggestion
    <p class="text-center note">Note: this procedure only helps migrate basic network settings to your Home Assistant SkyConnect, customizations like device names will be lost and automations will need to be updated. Most powered devices like lightbulbs will be re-discovered over time (you can speed this up by rebooting the device) but your battery-powered devices may need to be re-joined to the network for the migration to complete. There is currently no migration path to migrate all settings and devices.</p>
```